src/Plugin/Field/FieldFormatter/PlayerFormatter.php

```diff
@@ -329,6 +348,8 @@ private function addMediaFileDetails(array &$element, MpxMedia $mpx_media) {
    */
   private function buildPlayer(Media $source_plugin, Player $player, MpxMedia $mpx_media) {
     $url = new Url($source_plugin->getAccount(), $player, $mpx_media);
+    $url->setAutoplay((bool) $this->getSetting('auto_play'));
```

Comment:
Why do we need the casting here? Only thing I can think of would be if we defaulted these settings to `NULL`, allowing us to not include the parameters at all if they aren't set. However, that's not going to be possible as soon as you save the field formatter, which brings me back to the casting bit.

Reply:
You're right, the casting isn't needed. I think this was a relic from before I introduced the config schema and these values were coming through as strings, eg. `'1'` and `'2'`.